src/PIL/EpsImagePlugin.py

```diff
@@ -230,6 +230,11 @@ def _open(self) -> None:
         trailer_reached = False
 
         def check_required_header_comments() -> None:
+            """
+            The EPS spec requires that some headers exist.
```

Comment:
```suggestion
            The EPS specification requires that some headers exist.
```
Nitpick suggestion: Using the full word is clearer.